plugins/terraform/layer0/resource_layer0_load_balancer.go

```diff
@@ -8,6 +8,9 @@ import (
 
 	"github.com/hashicorp/terraform/helper/hashcode"
 	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/quintilesims/layer0/api/provider/aws"
+	"github.com/quintilesims/layer0/client"
+	"github.com/quintilesims/layer0/common/config"
 	"github.com/quintilesims/layer0/common/errors"
 	"github.com/quintilesims/layer0/common/models"
 )
```

Comment:
Ditto implementing `Exists` in `resourceLayer0LoadBalancer()`.

Reply:
not needed